### PR TITLE
composite-checkout: add Giropay payment method

### PR DESCRIPTION
**Pull request by sirbrillig**
#### Changes proposed in this Pull Request

This adds the Giropay Stripe redirect payment method to composite checkout.

#### Screenshots

Old checkout:

old-giropay

New checkout:

Unselected:

Screen Shot 2020-06-24 at 1 50 30 PM

Selected:

Screen Shot 2020-06-24 at 1 50 57 PM

#### Testing instructions

- Apply D45434-code to force Giropay to be available.
- Sandbox the store.
- Change your user's currency to `EUR`.
- Add a plan to your cart and visit checkout. 
- Add `?flags=composite-checkout-force` to the query string in the URL and reload the page because currently it is disabled for international currencies.
- Verify that "Giropay" is an option displayed in the final payment step.
- Verify that you cannot pay if the "Name" field is empty.
- Fill out the "Name" field and submit the payment.
- Verify that you are redirected to a Stripe test page.
- Click to accept the payment and verify that the payment completes successfully and that the plan is purchased.

### CODE REVIEW

**Comment by sirbrillig** [comment]
@fditrapani I'm not sure if you have any ideas about how to make the icon fit better into the radio button.

**Comment by fditrapani** [comment]
I updated the SVG for the logo so that it's 16px high. This is what it looks like now:

image

**Comment by sirbrillig** [comment]
Rebased on current main branch. Everything seems to be good, but let's double check, particularly a weird paypal logo regression we briefly saw during pairing.

**Comment by sirbrillig** [comment]
There was something weird going on with the giropay svg causing that box to hide the paypal logo if the giropay logo came first. I switched to using an img instead, which appears to have solved the issue.

File: client/my-sites/checkout/composite-checkout/wpcom/components/radio-button.js [hunk 1]
```diff
@@ -216,7 +223,7 @@ function handleLabelDisabled( { isDisabled } ) {
 			background: lightgray;
 		}
 		
-		:after {
+	:after {
```
Comment (nbloomf) [inline client/my-sites/checkout/composite-checkout/wpcom/components/radio-button.js:226]: I think this whitespace is off

**Comment by nbloomf** [comment]
Other than one very minor comment this looks good - we tested it quite a bit across two pairing sessions.

**Review by nbloomf — APPROVED** [review]

File: client/my-sites/checkout/composite-checkout/wpcom/components/radio-button.js (hunk 1 above)
Comment (sirbrillig) [reply client/my-sites/checkout/composite-checkout/wpcom/components/radio-button.js]: Weird. Thanks for noticing.

**Comment by sirbrillig** [comment]
Thanks! Test failures are the well-known Twitter ones.

**Pull request merged**